README_for_direct_archiver.md

```diff
@@ -11,49 +11,74 @@ Follow the steps:
     - prod - https://archiver.ingest.archive.data.humancellatlas.org
     
 2. Get the Archiver API key (`archiver_api_key`).
-```
-aws --region us-east-1 secretsmanager get-secret-value --secret-id ingest/archiver/wrangler/secrets --query SecretString --output text | jq -jr .{env}_archiver_api_key
-```
-Replace `{env}` with `dev`, `staging` or `prod` in the command above.
+   ```
+   aws --region us-east-1 secretsmanager get-secret-value --secret-id ingest/archiver/wrangler/secrets --query SecretString --output text | jq -jr .{env}_archiver_api_key
+   ```
+   Replace `{env}` with `dev`, `staging` or `prod` in the command above.
 
 3. Trigger the data archive request
-```
-curl -X POST {ingest_archiver_url}/archiveSubmissions/data -H 'Content-Type: application/json' -H "Api-Key:{archiver_api_key}" -d '{"sub_uuid": "{sub_uuid}"}'
-```
+   ```
+   curl -X POST {ingest_archiver_url}/archiveSubmissions/data -H 'Content-Type: application/json' -H "Api-Key:{archiver_api_key}" -d '{"sub_uuid": "{sub_uuid}"}'
```

Comment:
Why not use an environment variable for the submission uuid?

Reply:
@amnonkhen That's how @prabh-t wrote the code in the ingest-data-archive repository.
If you would like to change the code, 1st we have to change it there and not here.

Reply:
I meant to use environment variables instead of asking the user to replace stuff in the command line.

```suggestion
ingest_archiver_url=
archiver_api_key=
sub_uuid=
curl -X POST ${ingest_archiver_url}/archiveSubmissions/data -H 'Content-Type: application/json' -H "Api-Key:${archiver_api_key}" -d '{"sub_uuid": "${sub_uuid}"}'
```